doc/writeYourOwn/intro.rst

```diff
@@ -0,0 +1,18 @@
+Writing Your Own Cube Browser
```

Comment:
I know, this is really getting on my nerves and it's causing a lot of problems and is why I need to get some of these changes merged so that I can start making changes in series rather than in parallel, the latter of which tends to introduce small discrepancies which are confusing.